src/informacni-technologie/zakladni-desky-bios-a-efi/bios-a-efi.md

```diff
@@ -0,0 +1,42 @@
+# BIOS a EFI
+
+## BIOS
+
+- Basic Input/Output System
+- Firmware pro osobní počítače (základní rozhraní)
+- Původně od IBM - bylo nutno od nich kupovat kvůli kompatibilitě
+- Slouží k inicializaci a konfiguraci hardware v počítači a následnému spuštění operačního systému
+- Uložen je na základní desce v nevolatilní paměti (většinou v ROM - _Read-Only Memory_ - nebo modernější flash paměti)
+- V případě chyby při inicializaci systému na ni upozorní
+- **Možnosti nastavení:**
+  - Nastavení taktu procesoru, operační paměti
+  - Nastavení napětí procesoru
+  - Nastavení cache
+  - Detekce harddisků, CD-ROM
+  - Nastavení periferií
+  - Bootovací sekvence (HDD, CD-ROM, USB, LAN, FDD)
+  - HW monitor
+  - Power management
+
+## EFI
+
+- Extensible Firmware Interface
+- Nástupce BIOSu s otevřeným kódem
+- Původně vyvinuto společností Intel
+- V dnešní době nahrazeno UEFI (_Unified EFI_)
+- **Vlastnosti:**
+  - Podpora 64 bitových systémů
```

Comment:
Většina počítačů s Windows 7 používala legacy boot mode a většina z nich byla také 64 bit, takže to asi není specialita EFI.